indico/core/notifications.py

```diff
@@ -119,6 +120,31 @@ def flush_email_queue():
     db.session.commit()
 
 
+def _flush_webhook_queue():
+    """Send enqueued webhook notifications."""
+    from indico.modules.notifications.tasks import send_webhook_notification
+    queue = g.get('webhook_queue', set())
```

Comment:
no need for the default value since you'd return right below anyway if it's none

Reply:
can be changed for the email queue as well...